images.md

```diff
@@ -0,0 +1,22 @@
+# Lambda Instance images creation
+
+## Description
+
+Using the fokia library, an administrator is able to create updated images for the Lambda Instance Master and Slave nodes.
+
+## Usage
+
+1. To create the Lambda Instance images, one must use the fokia library. This can be done either on your own computer, after installing the required dependencies, or using an up-to-date Lambda Service VM.
```

Comment:
1. I guess I can not just install fokia on my mac and create a new image? It will have to be the system which will be used as the basis for the image, right?
2. How can one install fokia? Is this described somewhere else?